tl/features/context_match.py

```diff
@@ -148,7 +148,14 @@ def match_context_with_type(self, context: str, q_node: str, all_property_set: s
             p_value = prop[0]
             if context_data_type == 'q':
                 check_with_temp = p_value[1:]
-                check_with = float(check_with_temp.replace('"', ''))
+                check_with_temp = check_with_temp.replace('"', '')
+                # The following line handles cases q12wr or equivalent.
+                is_digit_list = [c.isdigit() or c == '.' for c in check_with_temp]
```

Comment:
very inefficient, you are checking each character in the string to be a digit. Has to be a better way

Reply:
Yes, there is I can add a try and except condition, that will be much faster. Is that okay?
Because there are chances of having something like q123a? so, if I have the except condition for the ValueError, there wouldn't be need to check each digit.

Reply:
So  the issue is coming from `i""hans\|quirin\|kaspar von sack""`, as I understand it, this is an `instance`. Shouldn't this be treated as string instead of quantity?

Reply:
Actually for it to be considered as an string, it should be of form: i""hans\|i""quirin\|i""kaspar von sack""
The reason why it is considered as quantity instead is because the differentiating variable is q after split with |.